src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,57 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.TreeMap;
+
+
+/**
+ * Class that takes a String as path file
+ * to csv files
+ * and creates datasets in two formats
+ * TreeMap & Table from tablesaw.
+ */
+
+public class CsvDataLoader {
+
+    /**
+     * Simple method that parses data set from a csv file
+     * without knowing the headers of the file
+     */
+    public void parseFile(String csvFile, TreeMap dataSet) throws IOException {
+
+        Reader csvData = new FileReader(csvFile);
+
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record : parser) {
+
+            for (String field : record) {
+                dataSet.keySet();
+                dataSet.get(field);
+
+                System.out.println("\"" + field + "\", ");
+            }
+        }
+    }
+
+    /**
+     * Reads from csv in file system with columns separated by commas
+     * and the file has a header row.
+     * Missing values are treated with an indicator.
+     */
```

Comment:
What do you think about this change? I think we will have to change the API to accommodate the change

Reply:
I agree. nice one! thanks.

Reply:
👍 👍  can you do the refactoring? The same has to be done for the JSON data. Also create a test case with valid json data sth in the spirit of ```dummy.csv``` I used

Reply:
will try to do that.

Reply:
nice